packages/cli/test/commands/new.test.ts

```diff
@@ -1,5 +1,56 @@
-// import {expect, test} from '@oclif/test'
+import ReadModel from '../../src/commands/new/read-model'
+import { IConfig } from '@oclif/config'
+import { restore, stub } from 'sinon'
+import * as fs from 'fs'
+import ErrnoException = NodeJS.ErrnoException
+import * as ProjectChecker from '../../src/services/project-checker'
+import { templates } from '../../src/templates'
+import Mustache = require('mustache')
+import { expect } from '../expect'
 
 describe('new', (): void => {
-  // TODO: Write proper tests
+  describe('Read model', () => {
+    const readModel = 'ExampleReadModel'
+    const readModelsRoot = './src/read-models/'
+    const readModelPath = `${readModelsRoot}${readModel}.ts`
+    afterEach(() => {
+      if (fs.existsSync(readModelPath)) {
+        fs.rmdir(readModelsRoot, { recursive: true }, (err: ErrnoException | null) => console.log(err))
+      }
+      restore()
```

Comment:
To be honest I'm not very convinced about this test (creating a file -> check input -> delete file). Suggestions are accepted 🙏